samples/overrides.d.ts

```diff
@@ -10,7 +10,7 @@ declare module overrides {
     toString(): string;
   }
 
-  export class B extends BLike {
```

Comment:
samples/overrides.d.ts(13,26): error TS2689: Cannot extend an interface 'BLike'. Did you mean 'implements'?